documentation/docs/contribute/development-guide/coding/01-architecture.md

```diff
@@ -95,25 +95,32 @@ We follow the [Clean Architecture](https://blog.cleancoder.com/uncle-bob/2012/08
 
 It is a layered approach for a [more civilized age](https://www.youtube.com/watch?v=wtCQalq7L-E).
 
+Clean architecture aims to divide a big project into layers and pieces (classes, entities, modules etc.), and to make them independent of each other, so that our big and complex project become more maintainable, flexible and easy to work on together.
```

Comment:
```suggestion
Clean architecture aims to divide a big project into layers and pieces (classes, entities, modules etc.), and to make them independent of each other, so that our big and complex project becomes more maintainable, flexible and easy to work on together.
```

Reply:
Fixed comment.